### PR TITLE
Merge Organization paired models

### LINKED ISSUE #549: Merge Organization and OrganizationExtra
OrganizationExtra provides:

* A `slug` field
* A `register` field for storing party registers (GB or NI). This is because there isn't a different `party` model – all conceptual "organisations" are stored in this model, including parties and bodies that hold elections.
* A GenericRelation to popolo.Image

This might link with making a dedicated Party model – work needs to be done to figure out the scope of this.

### PR DESCRIPTION
**Pull request by symroe**
You know the drill by now. 

This was a quick and easy one, as the Organization model is now only used in (more or less) the same way as we use them in EveryElection (things that have people elected to them).

I've also updated a few management commands that I think should be deleted, simply to help my find and replace work. I'll delete them in a future clean up PR, as I wanted to keep the scope of this limited (for a change).

Closes #549

### CODE REVIEW

**Comment by symroe** [comment]
[Merge Organization and OrganizationExtra YNR models](https://trello.com/c/brO0rcbv/1446-merge-organization-and-organizationextra-ynr-models)

File: ynr/apps/popolo/models.py [hunk 1]
```diff
@@ -744,6 +745,20 @@ class Organization(Dateframeable, Timestampable, models.Model):
         "Source", help_text="URLs to source documents about the organization"
     )
 
+    # Copied from OrganizationExtra
+    slug = models.CharField(max_length=256, blank=True, unique=True)
+    register = models.CharField(blank=True, max_length=512)
+    images = GenericRelation("images.Image")
+
+    def ec_id(self):
```
Comment (chris48s) [inline ynr/apps/popolo/models.py]: I know this isn't new code in this PR, but shouldn't this method have a special case for when I call it and the classification is not "Party" (e.g: classification = "local-authority")

File: ynr/apps/elections/uk/management/commands/uk_create_joint_pseudo_parties.py [hunk 2]
```diff
@@ -31,14 +29,11 @@ def create_or_update_party(joint_party_name, sub_parties):
     if len(registers) > 1:
```
Comment (chris48s) [inline ynr/apps/elections/uk/management/commands/uk_create_joint_pseudo_parties.py]: Just above here there is a block

```py
party_ids = sorted(
    extract_number_from_id(p.extra.slug) for p in sub_parties
)
registers = {p.extra.register for p in sub_parties}
```

should we also remove the `.extra` call from there?

**Comment by chris48s** [comment]
> This is needed because pytest is looking for files called `test_*.py`
> and wasn't picking up the file called `tests.py`.

That will mean there's a bunch of other tests that are being ignored then:

https://github.com/DemocracyClub/yournextrepresentative/blob/master/ynr/apps/popolo/tests.py
https://github.com/DemocracyClub/yournextrepresentative/blob/master/ynr/apps/popolo/behaviors/tests.py

We should probably delete these as well

https://github.com/DemocracyClub/yournextrepresentative/blob/master/ynr/apps/resultsbot/tests.py
https://github.com/DemocracyClub/yournextrepresentative/blob/master/ynr/apps/api/tests.py

just so we don't y'know.. put tests in them

On the upside, I expect this PR will increase our test coverage :D

File: ynr/apps/popolo/models.py (hunk 1 above)
Comment (symroe) [reply ynr/apps/popolo/models.py]: Fixed in 5223e1f

**Comment by symroe** [comment]
Ok, I think that's all points addressed now?

**Pull request merged**